kategory-optics/src/main/kotlin/kategory/optics/instances/NonEmptyListInstances.kt

```diff
@@ -0,0 +1,11 @@
+package kategory.optics
+
+import kategory.NonEmptyList
+
+/**
+ * [Lens] to operate on the head of a [NonEmptyList]
+ */
+fun <T> nelHead(): Lens<NonEmptyList<T>, T> = Lens(
```

Comment:
It would be worth adding it as an extfun to NEL's companion object.

Reply:
Should I do the same for the existing instances?

Reply:
Only the ones you have predefined, if you think it adds value! Definitely not a blocker.

Reply:
I am going to leave this open for now. I originally also wanted generated optics as extensions on their respective companion object but since this required the user to explicitly define companion objects everywhere I decided not to. I think adding them to companion objects in some cases and in some cases not is really confusing. Also then there is no way for the users to import all predefined optics `kategory.optics.*` or `kategory.optics.std.*`.